src/plugins/action/meson.build

```diff
@@ -0,0 +1,20 @@
+action_include = include_directories(['include'])
+
+# TODO: target properties
```

Comment:
Out of curiosity: what's that? :blush:

Reply:
> One question that I wonder: Would it not be easier to just use meson to build it as a cmake project?
@julianoes It might be, but for one, meson <-> cmake interop is not 100% bug free, and I thought it would be nicer to have a "pure" meson build system.

> Given it's easier to add support for meson like you did, I'm not opposed, however, a meson build needs to be added to CI. Otherwise we are inevitably going to break it.
If you can point me to what needs to be done for adding it to CI (I've never used Travis), I'd be happy to add a meson workflow.

Reply:
@JonasVautherin https://github.com/mavlink/MAVSDK/blob/develop/src/plugins/action/CMakeLists.txt#L10 Just some flags, nNot crucial to the build I think but I should add them in anyway :smiley: